lib/components/tabs/tabview.coffee

```diff
@@ -110,7 +110,10 @@ module.exports = class KDTabView extends KDScrollView
       minWidth : minHandleWidth
 
     newTabHandle.on 'HandleIndexHasChanged',  @bound 'resortTabHandles'
-    newTabHandle.closeHandler?.on 'click', => @handleCloseAction paneInstance
+
+    unless paneInstance.isDetached
+      newTabHandle.closeHandler?.on 'click', ->
+        paneInstance.parent.handleCloseAction paneInstance
```

Comment:
I need to get current / active tab view instance. Because when I change position of the tab by d&d, the tabview which attached a `click` event is remaining as wrong reference.